index.html

```diff
@@ -1,216 +1,285 @@
+          sign-ups required. Has been the industry's standard.
+        </p>
+        <ul>
+          <li class="list1">HTML</li>
+          <li class="list2">Bootstrap</li>
+          <li class="list3">Ruby</li>
+        </ul>
+        <button type="button" class="card-button">See project</button>
+      </article>
+      <article class="card">
+        <h5>Profesional Art Printing Data</h5>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. Has been the industry's standard.
+        </p>
+        <ul>
+          <li class="list1">HTML</li>
+          <li class="list2">Bootstrap</li>
+          <li class="list3">Ruby</li>
+        </ul>
+        <button type="button" class="card-button">See project</button>
+      </article>
+    </section>
+    <!-- about section -->
+    <section class="about">
+      <h2>About me</h2>
+      <p>
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hesitate to contact me.
+      </p>
+      <button type="button" class="about-button">Get my resume</button>
+    </section>
+    <div class="devider2">
+      <img src="Images/Vector 4.png" alt="vector4" />
+    </div>
+    <!-- skills section -->
+    <section class="skill">
+      <div class="skill-list">
+        <div class="frame1">
+          <img src="Images/Rectangle 55.png" alt="rectangle" />
+          <ul class="languages">
+            <li>Languages</li>
+          </ul>
+        </div>
+        <div class="frame2">
+          <ul>
+            <li>JavaScript</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </div>
+        <div class="frame1">
+          <img src="Images/Rectangle 56.png" alt="rectangle56" />
+          <ul class="frameworks">
+            <li>Frameworks</li>
+          </ul>
+        </div>
+        <div class="frame2">
+          <ul>
+            <li>React.js</li>
+            <li>Ruby on Rails</li>
+            <li>RSpec</li>
+            <li>Capybara</li>
+          </ul>
+        </div>
+        <div class="frame1">
+          <img src="Images/Rectangle 57.png" alt="rectangle57" />
+          <ul class="skills">
+            <li>Skills</li>
+          </ul>
+        </div>
+        <div class="frame2">
+          <ul>
+            <li>Database Management</li>
+            <li>Version Control</li>
+            <li>CLI</li>
+            <li>API Design</li>
+            <li>Web Development</li>
+          </ul>
+        </div>
+      </div>
+    </section>
+    <!-- contact form -->
+    <section class="contact-form">
+      <p>
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </p>
+      <form action="/submit" method="post">
```

Comment:
- [ ] You're required to collect the data from the form by using [Formspree Service](https://formspree.io/html/). Kindly go to that website, create an account to get your account_id. Then replace the form_id in the form with your account_id. 
change this :point_down: 
```<form action="https://formspree.io/f/{form_id}" method="post">```
  To
```<form action="https://formspree.io/f/{account_id}" method="post">```

![formspree](https://user-images.githubusercontent.com/26597432/228822018-aff9ae22-4509-44df-9515-9cf3fe0934f2.png)